apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.csproj

```diff
@@ -22,9 +22,9 @@
   </PropertyGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\Google.Cloud.Iam.V1\Google.Cloud.Iam.V1\Google.Cloud.Iam.V1.csproj" />
-    <PackageReference Include="ConfigureAwaitChecker.Analyzer" Version="1.0.0-beta4">
+    <!--<PackageReference Include="ConfigureAwaitChecker.Analyzer" Version="1.0.0-beta4">
       <PrivateAssets>All</PrivateAssets>
-    </PackageReference>
+    </PackageReference>--> <!-- TODO: Work out how to add this back -->
```

Comment:
How would I suppress the warning? There doesn't appear to be an warning number than I can use in a `#pragma warning disable`?